src/Tasks/Microsoft.Managed.targets

```diff
@@ -35,6 +35,16 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <DisableTransitiveProjectReferences Condition="'$(BuildProjectReferences)' != 'false' and '$(DisableTransitiveProjectReferences)' == ''">true</DisableTransitiveProjectReferences>
    </PropertyGroup>
 
+   <ItemGroup Condition="'$(IsGraphBuild)' == 'true'">
+      <!-- WPF projects generate a project with a random name at build time and then build the project via the IBuildEngine callbacks.
+           Detect WPF, and exclude the generated project from static graph isolation constraint checking.
+           Escape the item to avoid eager evaluation of the wildcards.
+      -->
+      <ReferencesToSkipGraphIsolationConstraintsOn
```

Comment:
I feel like there's a better name hiding here, but I don't have a great suggestion.

```
AllowedReference
IgnoredGraphIsolationReference
```

? I can live with this name if we can't find something clearly nicer.

Reply:
Alternatives
`ReferencesExemptedFromGraphIsolation`
`GraphIsolationExemptions`
`ReferencesExcludedFromGraphIsolation`

Reply:
`GraphIsolationExemptReference`?